### ISSUE TITLE
Missing WD Item for CT_222

### ISSUE THREAD

**Issue by djow2019** [SuLab/ChlamBase.org#56]
CT_222 is not found in WD even though its referenced here Q36281361
also update ortholog CTL_0475 to include CT_222

**Comment by andrewsu** [comment]
Looks like CT_222 was discontinued: https://www.ncbi.nlm.nih.gov/gene/884897  which is why there is no WD item.  On the scale of things, I think this is the correct behavior.  @khybiske can you have a look and if you agree, close the issue?

**Comment by andrewsu** [comment]
Yes, I think we should stay aligned with NCBI, but rather than waiting for users to complain I think we should proactively ask NCBI to review this case.  They have a form to request corrections here: 
https://www.ncbi.nlm.nih.gov/projects/RefSeq/update.cgi. @khybiske, want to fill this out or shall I? either way, we should point them back to this github issues so they have the ability to comment here (though it's also possible that it would only get an email response of course)...

**Comment by khybiske** [comment]
Before we ping NCBI, I want to hear back from a colleague (Dan Rockey) to get his two cents. Make sure I am not imagining something.

**Comment by khybiske** [comment]
@andrewsu What would we specify in the request form? Clarification on why the gene was discontinued? or request that it be reissued?

**Comment by andrewsu** [comment]
Both/either?  I think we want to propose a reexamination of that discontinuation, and hopefully they'd share with us the rationale.

**Comment by khybiske** [comment]
Sorry, I have not yet sent in the request. I'm dealing with a stack of grant and paper deadlines. As soon as the dust settles, I can do this.

**Comment by djow2019** [comment]
Has the request been sent?

**Comment by khybiske** [comment]
Ack, no.. I'll try to deal with it
